src/modelmanager.cpp

```diff
@@ -51,6 +51,7 @@
 namespace ovms {
 
 static bool watcherStarted = false;
+Status ModelManager::lastLoadConfigStatus = StatusCode::OK;
```

Comment:
In unit tests we inherit from ModelManager (ConstructorEnabledModelManager). This is static variable which may cause errors in unit tests when multiple run in parallel. Maybe add private field to ModelManager which would keep track of lastLoadConfigStatus?